raiden/network/transport/matrix/transport.py

```diff
@@ -1717,7 +1718,7 @@ def _receive_delivered_to_lc(self, delivered: Delivered):
         )
 
         assert self._raiden_service is not None
-        self._raiden_service.on_message(delivered, True)
+        self._raiden_service.on_message(delivered, self.get_address(), True)
```

Comment:
there is no point in keeping `get_address` around now that you have the `address` field

Reply:
sure, i will change that when the transport refactor is done